source/ImageBuffer.cpp

```diff
@@ -285,7 +283,7 @@ namespace {
 		
 		jpeg_stdio_src(&cinfo, file);
 		jpeg_read_header(&cinfo, true);
-		cinfo.out_color_space = JCS_EXT_BGRA;
+		cinfo.out_color_space = JCS_RGB;
```

Comment:
As discussed on discord; if libjpeg-turbo is working for all platforms, then what is preventing us from using JCS_EXT_RGBA directly (and avoiding the extra loop below)?

Reply:
Seems like that should work, I've just been looking at trying to measure load time instead. I make sure libjpeg-turbo works for the emscripten build.

Reply:
Ok

Reply:
Here's that much smaller change: https://github.com/endless-sky/endless-sky/pull/6155